src/ui/Assets/app.test.js

```diff
@@ -1,27 +1,23 @@
 import { initAll } from "govuk-frontend"
 import CookieMessage from "./Javascript/cookie-message"
 import BackLink from "./Javascript/back-link"
-import CharacterCount from "./Javascript/character-count"
 import serialize from "form-serialize"
 
 jest.mock("govuk-frontend")
 jest.mock("./Javascript/cookie-message")
 jest.mock("./Javascript/back-link")
-jest.mock("./Javascript/character-count")
 jest.mock("form-serialize")
 
 describe("App", () => {
   beforeAll(() => {
     document.body.innerHTML = `
-<div>
-  <div data-module="cookie-message"></div>
-  <div data-module="back-link"></div>
-  <div data-module="character-count"></div>
-  <div data-module="form"></div>
-  <div data-copy-course="warning"></div>
-</div>
-`
-
+      <div>
+        <div data-module="cookie-message"></div>
+        <div data-module="back-link"></div>
+        <div data-module="form"></div>
+        <div data-copy-course="warning"></div>
+      </div>
```

Comment:
Because this is a template string, it's fine for it to be at 0 indentation because the whitespace inside is part of the string, but there's no issue with aligning it with the code either (as long as we're not comparing strings for equality for example).

Reply:
Great tip 👌